lib/Test/Nginx/Util.pm

```diff
@@ -631,6 +632,18 @@ sub cleanup () {
     }
 }
 
+sub access_log_data () {
+    sleep $TestNginxSleep * 3;
+
+    open my $in, $AccLogFile or
+        return undef;
```

Comment:
We should distinguish the case of nonexistent file error and the other fatal error? We should not simply throw the system error message away, which will make debugging very difficult when there is something unusual happens when trying to open the access log file.